app/covidbed/serializer/resource.py

```diff
@@ -0,0 +1,74 @@
+from flask_restful_swagger import swagger
+from flask_restful import fields
+
+
+@swagger.model
+class HumanSerializer:
+    resource_fields = {
+        'reanimators_count': fields.Integer,
+        'need_staff': fields.Integer,
+    }
+
+
+@swagger.model
+class BedSerializer:
+    resource_fields = {
+        'covid_available': fields.Integer,
+        'covid_used': fields.Integer,
+        'other_available': fields.Integer,
+        'other_used': fields.Integer,
+        'conventional_count': fields.Integer,
+        'continue_care_count': fields.Integer,
+        'reanimation_count': fields.Integer,
+        'post_urgency_count': fields.Integer,
+        'other_count': fields.Integer,
+    }
+
+
+@swagger.model
+class SupplySerializer:
+    resource_fields = {
+        'reanimators_count': fields.Integer,
+        'efp2_mask_count': fields.Integer,
```

Comment:
```suggestion
        'efp2_masks_count': fields.Integer,
```
Maybe we need consistency with the plurals.

Reply:
done